src/lib/forward.ts

```diff
@@ -72,13 +67,14 @@ export class EventForwarder {
       })
     );
 
-    this.clients.driver.controller.on("node removed", (node) =>
+    this.clients.driver.controller.on("node removed", (node, replaced) =>
       // forward event to all connected clients, respecting schemaVersion it supports
       this.clients.clients.forEach((client) =>
         this.sendEvent(client, {
           source: "controller",
           event: "node removed",
           node: dumpNode(node, client.schemaVersion) as any,
+          replaced,
```

Comment:
How about this, similar to the `result` in `"node added"`?

```suggestion
          "replaced": replaced,
```

Reply:
We only do that because we have to cast the `result` as `Any`. In this case, `replaced` is shorthand for `replaced: replaced`

Reply:
Didn't know that, thanks!